cmd/subnetcmd/export.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package subnetcmd
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/application"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/spf13/cobra"
+)
+
+var exportOutput string
+
+// avalanche subnet list
+func newExportCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export [subnetName]",
+		Short: "Export deployment details",
+		Long: `The subnet export command prints the details of an existing subnet deploy.
+
+The command prompts for an output filename. You can also provide one with
+the --output flag.`,
+		RunE:         exportSubnet,
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
+	}
+
+	cmd.Flags().StringVarP(
+		&exportOutput,
+		"output",
+		"o",
+		"",
+		"write the export data to the provided file path",
+	)
+
+	return cmd
+}
+
+func exportSubnet(cmd *cobra.Command, args []string) error {
+	var err error
+	if exportOutput == "" {
+		pathPrompt := "Enter file path to write export data to"
+		exportOutput, err = app.Prompt.CaptureString(pathPrompt)
```

Comment:
do we need to check for valid path?

Reply:
I think this is probably fine, if the user messes up, they can re-enter easily. If we want to in the future, we can add a function that checks the permissions of the directory, but I think that's a little much for right now. I'd want to test that more than we have time for right now.